src/main/java/uk/gov/pay/api/service/GetPaymentEventsService.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.pay.api.service;
+
+import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.exception.GetEventsException;
+import uk.gov.pay.api.model.PaymentEvents;
+import uk.gov.pay.api.model.PaymentEventsResponse;
+
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+
+import static org.apache.http.HttpStatus.SC_OK;
+
+public class GetPaymentEventsService {
```

Comment:
Can we not use `ConnectorService` to get Payment events from connector. I feel ConnectorService is set for this purpose  or I am happy if the plan is to use ConnectorService eventually

Reply:
there will be further refactoring to use `ConnectorService`